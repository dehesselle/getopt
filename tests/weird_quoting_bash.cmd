#!/bin/sh
"$1" -s bash -o a: -l long1: -- -a '$?!* ' --long \"\\\' 'More
 than
one 
line'
