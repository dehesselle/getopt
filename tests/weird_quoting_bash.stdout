 -a '$?!* ' --long1 '"\'\''' -- 'More
 than
one 
line'
